src/applications/check-in/api/local-mock-api/e2e/ApiInitializer.js

```diff
@@ -23,6 +23,9 @@ class ApiInitializer {
           checkInExperienceEnabled: true,
           preCheckInEnabled: true,
           emergencyContactEnabled: true,
+          checkInExperienceEditingPreCheckInEnabled: false,
```

Comment:
Should this Edit toggle be here? I think this may have snuck its way back in.

Reply:
nope! good catch, thanks :-)